example/lib/main.dart

```diff
@@ -47,7 +47,7 @@ class _MyHomePageState extends State<MyHomePage> {
           children: <Widget>[
             Text(
               'You have pushed the button this many times:',
-              style: GoogleFonts.oswald(textStyle: display1),
+              style: GoogleFonts.getFont("Oswald", textStyle: display1),
```

Comment:
nit: use single quotes for strings

Reply:
Actually, I don't think this should be changed here. We would prefer users to use `GoogleFonts.oswald` over `GoogleFonts.getFont('Oswald'` unless that had to use the string based API